styles.css

```diff
@@ -629,3 +633,183 @@ textarea::placeholder {
   font-size: 15px;
   line-height: 24px;
 }
+
+.tonic-1 {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    max-width: 100%;
+  }
+
+  .dsk-header-logo {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 20px;
+    background-color: #fff;
+    padding-top: 1.875rem;
+  }
+
+  #body-cover {
+    background-image: url('images/Header\ bgDesktop.svg');
+    background-size: cover;
+    padding-bottom: 21rem;
+    border-left: 2rem solid white;
+    min-height: 30rem;
+    border-bottom: 6rem solid white;
+  }
+
+  .image-icon {
+    display: none;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  .logo-2 {
+    margin-left: 10rem;
+  }
+
+  .intro {
+    margin-left: 16.25rem;
+    margin-right: 16.25rem;
+    max-width: 45.5rem;
+  }
+
+  .lets-connect {
+    margin-left: 16.25rem;
+  }
+
+  .social-buttons {
+    margin-left: 16.25rem;
+    margin-top: 1rem;
+  }
+
+  .nav-content {
+    list-style-type: none;
+    display: flex;
+    justify-content: space-between;
+    padding-right: 172px;
+    align-items: center;
+    gap: 86px;
+  }
+
+  .title {
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    margin-top: 9.438rem;
+    margin-left: 16.25rem;
+    margin-right: 16.25rem;
+  }
+
+  .works {
+    /* display: flex;
+    flex-direction: row; */
+    display: grid;
+    grid-template-columns: 0.5fr 1fr;
+    max-width: 75%;
+    margin-left: 15rem;
+  }
+
+  .works2 {
+    grid-template-columns: 0.5fr 1fr reverse;
+  }
+
+  .tonic-1 {
+    display: block;
+    padding: 24px 24px 24px;
+  }
+
+  .tonic {
+    display: none;
+  }
+
+  .header-tonic {
+    padding-top: 20px;
+  }
+
+  .stories {
+    padding-top: 20px;
+  }
+
+  .myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    height: auto;
+  }
+
+  .languages-below {
+    padding-top: 50px;
+    margin-left: 40px;
+  }
+
+  .language-icons {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .language-icons li {
+    display: flex;
+    flex-direction: column;
+  }
+
+  ul.description {
+    display: flex;
+    align-items: center;
+    justify-content: normal;
+    margin-left: 2%;
+    margin-right: 150px;
+    gap: 1rem;
+  }
+
+  .contact-form {
+    background-image: url('images/contact\ form\ background\ shapesdesktop.svg');
+    padding: 20px;
+  }
+
+  .form-1 {
+    padding-left: 320px;
+    margin-left: 389px;
+    margin-top: 142px;
+  }
+
```

Comment:
- [ ] Your contact form was designed beautifully with the right background images 👏  However, kindly adjust it to be centered just ass given on the figma template. With the suggested changes, it will give a more appealing view to the user with translates to a better experience. Images below for your review 

## Your design:
![image](https://user-images.githubusercontent.com/50990925/222940340-6fb8744b-6cfa-4874-9fd0-5bcb6bcc04d1.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/222940347-a955cbf4-31f0-4ab5-b369-57d5e988cbe5.png)